CoreNLG/decorators.py

```diff
@@ -77,6 +77,23 @@ def handle_dots(text):
     return text
 
 
+def handle_redundant_poncts(text, ponct):

```

Comment:
Maybe you should the full word *ponctuation* instead of using the contracted form just to be clear. I toot a minute to figure that poncts <=> ponctuations